packages/p2p/src/pages/my-ads/ad-conditions-section/ad-conditions-section.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { FormikValues, useFormikContext } from 'formik';
+import { Button, Icon, Popover, Text } from '@deriv/components';
+import BlockSelector from 'Components/block-selector';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import AdFormController from 'Pages/my-ads/ad-form-controller';
+import { useStores } from 'Stores';
+import CreateAdSummary from '../create-ad-summary.jsx';
+
+const AdConditionsSection = ({ ...props }) => {
+    const { errors, isSubmitting, isValid, values } = useFormikContext<FormikValues>();
+    const { showModal } = useModalManagerContext();
+    const { my_ads_store } = useStores();
+    const { min_completion_rate, min_join_days } = my_ads_store.p2p_advert_information;
+    const joining_days = [
+        { name: '15 days', value: 15 },
+        { name: '30 days', value: 30 },
+        { name: '60 days', value: 60 },
+    ];
+    const completion_rates = [
+        { name: '50%', value: 50 },
+        { name: '70%', value: 70 },
+        { name: '90%', value: 90 },
+    ];
```

Comment:
can we move these out of the component?

Reply:
These are only being used in this component.

Reply:
yeah i mean outside of the component, like above it so it doesn't get rerendered 😄